doc/content/reference/webhook-templates/format.md

```diff
@@ -0,0 +1,54 @@
+---
+title: "Format"
+description: ""
+weight: 2
+---
+
+Webhook templates are described using the [YAML](https://yaml.org/) language. Their format is very closely related to that of a normal webhook integration, but with additional fields added.
+
+## Service Description
+
+All of the webhook templates must contain the following fields which describe the service provided by the template to the user.
+
+- `applicationwebhooktemplateidentifiers.templateid`: The unique identifier of the template.
```

Comment:
I think we have to do two things;
- Get rid of this long thing
- Introduce dashes in keys

Would this break configuration? Did we release this stuff at all?

Reply:
I think that the long part (`applicationwebhooktemplateidentifiers`) is there for the RPC validation.
Regarding the dashes - maybe it's an option in the YAML de/serializer ? I'll check.

Reply:
How so RPC validation?

Reply:
Nevermind - I thought we have (only) the identifiers as part of the `GET` request. We can simply use the validated string instead. Will add.